src/pages/[teamName]/[projectName]/[testId].tsx

```diff
@@ -229,34 +236,44 @@ const TestPage = withError<GET_SERVER_SIDE_PROPS_ERROR, ITestProps>(
                 fontWeight={900}
                 fontSize="xl"
               >
-                Test Failures
-                <Code
-                  ml={2}
-                  fontSize="inherit"
-                  variantColor={
-                    status === "In progress"
-                      ? "yellow"
-                      : status === "Passing"
-                      ? "cyan"
-                      : status === "Failed"
-                      ? "red"
-                      : null
-                  }
-                >
-                  {location}@
-                  <Link
-                    href={`https://github.com/${teamName}/${projectName}/commit/${commitHash}`}
+                {failures.length} Test Failure
+                {failures.length === 1 ? null : "s"}
```

Comment:
```suggestion
                {failures.length === 1 ? "Test Failure" : "Test Failures"}
```

Reply:
@mikesol Are you not wanting to display the count of failures then?